### DISCUSSION TITLE
Standarize the mikrotik names or models

### DISCUSSION THREAD

**Discussion by RataDP** [netbox-community/devicetype-library#1086] [Polls]
Hello,

I started to use Netbox. In our company almost every device are mikrotiks. 
Looking the mikrotiks devices listed in the library, I see there are names that are the model (ex. RB750r3) but its commercial name is hEX and others viceversa, using the commercial name as name in netbox.

I guess this will need to be standardized. I want to do a PR with several mikrotiks but I don't know if I has to use the comercial name or the model. In my use case or my company, we always call the mikrotik by its commercial name: hex, hap, ltap, netbox (if they have one)

If the commercial name is the name of the device, the model should be the PN of the device

**Comment by danner26** [comment]
Hey @RataDP I am one of the newer maintainers on this repo and one of my undertakings has been to cleanup and standardize the data in here. I would personally like this to be as accurate as possible and there is a ton of data that could be improved upon
Are you in the [NetBox Slack](https://netdev.chat/)? If so, maybe you could private message me to work on this together. My username in there is @danner26

**Reply by danner26** [reply]
So looking into this more, for products like the hEX I would do:
```
model: hEX
part_number: RB750Gr3
```

This is because according to the MikroTik page, the name of the device is hEX (aka model), and the SKU or Product Code is RB750Gr3
